src/MISC/fix_deposit.cpp

```diff
@@ -11,10 +11,10 @@
    See the README file in the top-level LAMMPS directory.
 ------------------------------------------------------------------------- */
 
-#include "fix_deposit.h"
```

Comment:
this is undoing recent changes for increased consistency of include file statements. please restore to the original.